client/python/gradio_client/utils.py

```diff
@@ -311,7 +313,7 @@ async def get_pred_from_ws(
                             # otherwise will get nasty warning in console
                             task.cancel()
                             await asyncio.gather(task, reset, return_exceptions=True)
-                        raise CancelledError()
+                        raise concurrent.futures.CancelledError()
```

Comment:
Why this change?

Reply:
This is just because I've replaced

```
from concurrent.futures import CancelledError
```

with 

```
import concurrent.futures
```